src/hint_processor/builtin_hint_processor/secp/bigint_utils.zig

```diff
@@ -0,0 +1,413 @@
+}
+
+test "BigIntN Hints: get bigint3 from var name with missing member should fail" {
+    var vm = try CairoVM.init(std.testing.allocator, .{});
+
+    defer vm.deinit();
+
+    try vm.segments.memory.setUpMemory(std.testing.allocator, .{
+        .{ .{ 1, 0 }, .{1} },
+        .{ .{ 1, 1 }, .{2} },
+    });
+
+    defer vm.segments.memory.deinitData(std.testing.allocator);
+
+    vm.run_context.fp.* = Relocatable.init(1, 1);
+
+    var ids_data = try testing_utils.setupIdsForTestWithoutMemory(std.testing.allocator, &.{"x"});
+    defer ids_data.deinit();
+
+    var x: BigInt3 = undefined;
+
+    try std.testing.expectError(MemoryError.UnknownMemoryCell, x.fromVarName("x", &vm, ids_data, .{}));
+}
+
+test "BigIntN Hints: get bigint5 from var name with missing member should fail" {
+    var vm = try CairoVM.init(std.testing.allocator, .{});
+
+    defer vm.deinit();
+
+    try vm.segments.memory.setUpMemory(std.testing.allocator, .{
+        .{ .{ 1, 0 }, .{1} },
+        .{ .{ 1, 1 }, .{2} },
+        .{ .{ 1, 2 }, .{3} },
+        .{ .{ 1, 3 }, .{4} },
+    });
+
+    defer vm.segments.memory.deinitData(std.testing.allocator);
+
+    vm.run_context.fp.* = Relocatable.init(1, 1);
+
+    var ids_data = try testing_utils.setupIdsForTestWithoutMemory(std.testing.allocator, &.{"x"});
+    defer ids_data.deinit();
+
+    var x: BigInt5 = undefined;
+
+    try std.testing.expectError(MemoryError.UnknownMemoryCell, x.fromVarName("x", &vm, ids_data, .{}));
+}
+
+test "BigIntN Hints: get bigint3 from varname invalid reference should fail" {
+    var vm = try CairoVM.init(std.testing.allocator, .{});
+
+    defer vm.deinit();
+
+    try vm.segments.memory.setUpMemory(std.testing.allocator, .{
+        .{ .{ 1, 0 }, .{1} },
+        .{ .{ 1, 1 }, .{2} },
+        .{ .{ 1, 2 }, .{3} },
+    });
+
+    defer vm.segments.memory.deinitData(std.testing.allocator);
+
+    var ids_data = try testing_utils.setupIdsForTestWithoutMemory(std.testing.allocator, &.{"x"});
+    defer ids_data.deinit();
+
+    var x: BigInt3 = undefined;
+
+    try std.testing.expectError(HintError.UnknownIdentifier, x.fromVarName("x", &vm, ids_data, .{}));
+}
+
+test "BigIntN Hints: get bigint5 from varname invalid reference should fail" {
+    var vm = try CairoVM.init(std.testing.allocator, .{});
+
+    defer vm.deinit();
+
+    try vm.segments.memory.setUpMemory(std.testing.allocator, .{
+        .{ .{ 1, 0 }, .{1} },
+        .{ .{ 1, 1 }, .{2} },
+        .{ .{ 1, 2 }, .{3} },
+        .{ .{ 1, 3 }, .{4} },
+        .{ .{ 1, 4 }, .{5} },
+    });
+
+    defer vm.segments.memory.deinitData(std.testing.allocator);
+
+    var ids_data = try testing_utils.setupIdsForTestWithoutMemory(std.testing.allocator, &.{"x"});
+    defer ids_data.deinit();
+
+    var x: BigInt5 = undefined;
+
+    try std.testing.expectError(HintError.UnknownIdentifier, x.fromVarName("x", &vm, ids_data, .{}));
+}
+
+test "Run hiMaxBitlen ok" {
```

Comment:
Hey @tcoratger , only this test is failing, all the other tests are working fine. Would it be possible for you help here?

ps - With this fix, the PR should be ready to merge